index.html

```diff
@@ -15,8 +15,8 @@ <h1 class="logo__text">Renome</h1>
                     </a>
                     <div class="header__container">
```

Comment:
i think parent class name should be just header and under the parent will go all childes like __container, __card